tests/group_vars/vagrant.yml

```diff
@@ -73,16 +73,16 @@ dataverse:
       enabled: false
   filesdir: /usr/local/dvn/data
   glassfish:
-    user: payara
-    group: payara
+    user: glassfish
+    group: glassfish
     domain: domain1
     logformat: ulf
     adminuser: admin
     adminpass: notPr0d
     timeout: 180
     siteurl: 
     root: /usr/local
-    dir: payara5
+    dir: glassfish4
```

Comment:
Does the back and forth between glassfish and payara mean that both are supported by dataverse-ansible when running the python installer? If so, that's really cool! If not, please don't worry about it. So far, after my switch to payara for dev I haven't looked back.

Reply:
correct, zipurl and several other glassfish/payara group_vars were there to allow testing, and a few logic switches based on glassfish4/payara5 happen as needed.